staging/src/k8s.io/apiserver/pkg/storage/OWNERS

```diff
@@ -19,3 +19,5 @@ emeritus_approvers:
   - xiang90
   - timothysc
   - lavalamp
+labels:
+  - sig/etcd
```

Comment:
this is technically kube-apiserver which is owned by api-machinery.

Reply:
moved this under `staging/src/k8s.io/apiserver/pkg/storage/etcd3`

Reply:
will wait for sig-apimachinery review